### DISCUSSION TITLE
Filenames

### DISCUSSION THREAD

**Discussion by wanneut** [nfdi4plants/ARC-specfication#25] [Comments and Suggestions]
I would like to rewrite the "Directory and File Naming Conventions" section and use the terms Unicode Character Categorie Naming Conventions instead of examples. This will make this section more specific.
But at first I have an other Question:
Do we really want to limit filenames to ASCII? There is no OS or Filesystem that can not handle at least the full UCS2-character set since 20 years . But a lot of filesystems with different restrictions of path depth and length but we don't have restrictions for that kind of Struff

**Comment by muehlhaus** [comment]
Thank you for the discussion point. I personally agree fully. I would get rid of such restrictions, if there is no important technical reason (that I don't see).

**Comment by dvs-frbg** [comment]
Open for good alternatives, suggestions! Sometimes we see strange behavior of tools improperly handling filenames in other contexts; just a precaution, but not necessary. Human readability would be a plus. Good point with the total length of paths (see discovered Linux kernel vulnerability)

**Reply by chgarth** [reply]
Agree, but this is placing an arguably non-trivial a burden on user, so I would avoid unless there is a specific need and not just a generic concern. I am also not sure that any specifications we make could a) be compatible with all use cases (ARCs on DOS floppies?) and b) will work in the future if a hot new OS / filesystem should be used. 

(From a purely technical point of view, ARCs are really Git repos, so OS/FS problems appear only when a checkout is made, which could be worked around in really problematic scenarios.)

**Reply by chgarth** [reply]
My suggestion would be to completely remove the "Directory and File Naming Conventions" section.
